frontend/online-shop/src/theme/MyTheme.tsx

```diff
@@ -0,0 +1,19 @@
+import {createMuiTheme} from '@material-ui/core/styles'
+import '../styles/Fonts.css'
+
+
```

Comment:
Here an empty line is required but never use two

Reply:
Fixed.